.github/workflows/ci.yml

```diff
@@ -8,11 +8,9 @@ name: ci
 # trigger on pull_request or push events
 on:
   push:
-    branches:
```

Comment:
yaml validation didn't like this and this change would make this action consistent with what we do elsewhere, eg. https://github.com/go-vela/server/blob/ea8fadd3c76883df252fb3c3b60681deebd85e79/.github/workflows/test.yml#L4-L7